### PR TITLE
User and badge chat commands

### PR DESCRIPTION
**Pull request by Felk**
requires https://github.com/TwitchPlaysPokemon/tpp-core/pull/68 to be merged first

### CODE REVIEW

File: ArgsParsing/TypeParsers/UserParser.cs [hunk 1]
```diff
@@ -26,9 +27,12 @@ public UserParser(IUserRepo userRepo)
         public override async Task<ArgsParseResult<User>> Parse(IImmutableList<string> args, Type[] genericTypes)
```
Comment (red031000) [inline ArgsParsing/TypeParsers/UserParser.cs:27]: this should also accept display names

File: Core/Chat/TwitchChat.cs [hunk 2]
```diff
@@ -19,7 +19,9 @@ namespace Core.Chat
     public sealed class TwitchChat : IChat
     {
         public event EventHandler<MessageEventArgs> IncomingMessage = null!;
-        private static readonly MessageSplitter MessageSplitter = new MessageSplitter(maxMessageLength: 500);
+        private static readonly MessageSplitter MessageSplitterRegular = new MessageSplitter(maxMessageLength: 500);
+        private static readonly MessageSplitter MessageSplitterWhisper = new MessageSplitter(
+            maxMessageLength: 500 - "/w ,,,,,''''',,,,,''''',,,,, ".Length);
```
Comment (red031000) [inline Core/Chat/TwitchChat.cs:24]: ??

File: ArgsParsing/TypeParsers/UserParser.cs (hunk 1 above)
Comment (Felk) [reply ArgsParsing/TypeParsers/UserParser.cs:27]: I know, I deliberately do not want to do that myself as that's a great first issue for someone else

File: Core/Chat/TwitchChat.cs (hunk 2 above)
Comment (Felk) [reply Core/Chat/TwitchChat.cs]: I reverted the changes in this branch

**Pull request merged**